docs_src/getting-started/Ch-GettingStartedHybrid.md

```diff
@@ -26,63 +24,72 @@ EdgeX environment up and running via Docker containers. How would you set up thi
 
-Examine the **configuration.toml** file in the cmd/res folder of the device-virtual-go. Note that the Registry (located in the \[Registry\] section of the configuration) and all the "clients" (located in the \[clients\] section of the configuration file) suggest that the "Host" of these services is "localhost".  These and other host configuration elements need to change when the services are not running on the same host.  If you do have to change the configuration, save the configuration.toml file after making changes.
+Examine the **configuration.toml** file in the cmd/res folder of the device-virtual-go. Note that the Service (located in the \[Service\] section of the configuration), Registry (located in the \[Registry\] section) and all the "Clients" (located in the \[Clients\] section) suggest that the `Host` of these services is "localhost".  These and other host configuration elements need to change when the services are not running on the same host - specifically the localhost.  When your service is running on a different host than the rest of EdgeX, change the \[Service\] `Host` to be the address of the machine hosting your service.  Change the \[Registry\] and \[Clients\] `Host` configuration to specify the location of the machine hosting these services.  If you do have to change the configuration, save the configuration.toml file after making changes.
 
 ### Run the service code natively.  
 
-The executable created by the make command is usually found in the cmd folder of the service.
+The executable created by the `make build` command is found in the cmd folder of the service.  Change folders to the location of the executable.  Set any environment variables needed depending on your EdgeX setup.  In this example, we did not start the security elements so we need to set `EDGEX_SECURITY_SECRET_STORE` to `false` in order to turn off security.   Finally, run the service right from a terminal.
+
 ``` bash
 cd cmd
+export EDGEX_SECURITY_SECRET_STORE=false
 ./device-virtual
 ```
 
 ![image](EdgeX_GettingStartedHybridRun.png)
+*Change folders to the service's cmd/ folder, set env vars, and then execute the service executable in the cmd folder.*
 
 ### Check the results
 
-At this time, your virtual device micro service should be communicating with the other EdgeX micro services running in
-their Docker containers. Give the virtual device a few seconds or so to
-initialize itself and start sending data to Core Data. To check that it
-is working properly, open a browser and point your browser to Core Data
-to check that events are being deposited. You can do this by calling on
-the Core Data API that checks the count of events in Core Data
-http://[host].48080/api/v1/event/count.
+At this time, your virtual device micro service should be communicating with the other EdgeX micro services running in their Docker containers. Give the virtual device a few seconds or so to
```

Comment:
Need a note that Core Metadata callbacks do not work in hybrid environment. Device Virtual will not receive the Add Device callbacks on the initial run after creating them in Core Metadata. The simple work around it to restart Device Virtual which will then we see the devices already exist and use them. Could also add an image of the log message after restarting.